hazelcast/src/test/java/com/hazelcast/internal/cluster/impl/AddressCheckerImplTest.java

```diff
@@ -77,6 +77,30 @@ public void givenInterfaceRangeIsConfigured_whenMessageWithNonMatchingHost_thenD
         assertFalse(joinMessageTrustChecker.isTrusted(address));
     }
 
+    @Test
+    public void testAsteriskWildcard() throws UnknownHostException {
+        AddressCheckerImpl joinMessageTrustChecker = new AddressCheckerImpl(singleton("127.0.*.*"), logger);
+        assertTrue(joinMessageTrustChecker.isTrusted(createAddress("127.0.1.1")));
+        assertFalse(joinMessageTrustChecker.isTrusted(createAddress("127.1.1.1")));
+
+        joinMessageTrustChecker = new AddressCheckerImpl(singleton("127.*.1.*"), logger);
+        assertTrue(joinMessageTrustChecker.isTrusted(createAddress("127.0.1.1")));
+        assertTrue(joinMessageTrustChecker.isTrusted(createAddress("127.127.1.127")));
+        assertFalse(joinMessageTrustChecker.isTrusted(createAddress("127.127.127.127")));
+    }
+
+    @Test
+    public void testIntervalRange() throws UnknownHostException {
+        AddressCheckerImpl joinMessageTrustChecker = new AddressCheckerImpl(singleton("127.0.110-115.*"), logger);
+        assertTrue(joinMessageTrustChecker.isTrusted(createAddress("127.0.110.1")));
```

Comment:
Could you please add also test for:
- number which is inside of boundaries (for example `127.0.112.1`)
- number from upper boundary (for example `127.0.115.1`)